app/scripts/restore-dump.ts

```diff
@@ -12,25 +12,8 @@ const argv = await yargs(hideBin(process.argv)).option("dump-file-path", {
 const dbName = "openpipe-dev";
 const templateDbName = `${dbName}_template`;
 
-const migrateTemplate = `
```

Comment:
We can just delete this file I think; it was only relevant for the Render dumps that we aren't using anymore.